src/utils/helpers/date/date.js

```diff
@@ -99,8 +116,13 @@ const DateHelper = {
    * @param {String} units - defaulted to days
    * @return {Boolean}
    */
-  withinRange: (value, limit, units) => {
```

Comment:
Please document this as a breaking change.

Reply:
added, thanks